app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,186 @@
+    <string name="pref_query_hide_root_title">Nascondi processi di root</string>
+    <string name="pref_query_hide_root_summaryOn">I processi di root non saranno mostrati.</string>
+    <string name="pref_load_avg_title">Tempo di caricamento medio</string>
+    <string name="pref_load_avg_summary">Periodo di tempo utilizzato per calcolare la media di carico.</string>
+    <string name="pref_query_show_system_time_title">Mostra il tempo di sistema</string>
+    <string name="pref_query_show_system_time_summaryOff">Il tempo di sistema è nascoto.</string>
+    <string name="pref_query_show_system_time_summaryOn">Il tempo di sistema è visualizzato.</string>
+    <string name="pref_log_title">Visualizza log</string>
+    <string name="pref_changelog_title_summary">Visualizza changelog dell\'applicazione</string>
+    <string name="pref_log_title_summary">Visualizza log dell\'applicazione</string>
+    <string name="changelog_title">News</string>
+    <string name="pref_debug_log_summary_off">Livello di log debug disabilitato.</string>
+    <string name="pref_version">Versione</string>
+    <string name="pref_screen_about_summary">Versione, changelog, sviluppatore</string>
+    <string name="pref_changelog_title">Visualizza changelog</string>
+    <string name="pref_rate_app">Valuta l\'app</string>
+    <string name="pref_github">Sorgemto / Issue Tracker</string>
+    <string name="pref_dev_email">Contatta lo sviluppatore</string>
+    <string name="changelog_full_title">Changelog</string>
+    <string name="network_link">Qualità</string>
+    <string name="please_add_a_raspberry_pi">Per favore aggiungi un Raspberry PI.</string>
+    <string name="no_device_available">Per favore prima aggiungi un Raspberry PI.</string>
+    <string name="dialog_passphrase_text">Per favore inserisci un passphrase per la private key.</string>
+    <string name="validation_msg_keyfile">Per favore selezione un keyfile.</string>
+    <string name="no_keyfile_present">Per favore seleziona una private key esistente.</string>
+    <string name="new_raspi_minimum">Per favore specifica almeno nome, host, username ssh e password.</string>
+    <string name="no_connection">Per favore attiva il WiFi.</string>
+    <string name="pref_debug_log_summary">Stampa messagi di debug su log.</string>
+    <string name="raspi_auth_keyfile_hint">Private key (formato PKCS8/OpenSSH)</string>
+    <string name="title_placeholders">Segnaposti</string>
+    <string name="update_successful">PI aggioranto con successo.</string>
+    <string name="new_pi_created">PI creato.</string>
+    <string name="permission_private_key_error">Impossibile leggere la primate key. Impossibile eseguire la query.</string>
+    <string name="dialog_passphrase_needed">Passphrase necessaria</string>
+    <string name="ssh_credentials">Credenziali SSH</string>
+    <string name="ssh_credentials">Credenziali SSH</string>
+    <string name="ssh_credentials">Credenziali SSH</string>
```

Comment:
String `ssh_credentials` is included three times, can you fix it?

Reply:
I only checked values/strings.xml (default english), there's no duplication there.